scripts/lint.py

```diff
@@ -47,8 +54,14 @@ def main(args):
   os.chdir(settings.GetRoot())
   try:
     cl = git_cl.Changelist(auth_config=auth_config)
-    change = cl.GetChange(cl.GetCommonAncestorWithUpstream(), None)
-    files = [f.LocalPath() for f in change.AffectedFiles()]
+    if not options.base_branch:
+      print("Set the base branch in lib/config.js")
+      return 0
+    if not options.base_branch:
+      print("Set the current brave_core branch in package.json")
+      return 0
+    change = git_cl.RunGit(['diff', '--name-only', options.current_branch, options.base_branch])
```

Comment:
actually just noticed these changes and I'm a bit concerned by them. Why wouldn't we still just call `cl.GetChange` and just pass something else in place of `cl.GetCommonAncestorWithUpstream()`?